### DISCUSSION TITLE
Firefox Addon Stopped working.

### DISCUSSION THREAD

**Discussion by HALLANM** [aclap-dev/video-downloadhelper#584] [Help and Q&A]
### Your question

 New version of both Add-on and Co-App did not help.  It stopped working BEFORE I updated the Co-APP

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://iconnectfx.com/view/54e72282-6617-ef11-996f-0050568299de/en",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : dyld: cannot load 'ffmpeg' (load command 0x80000034 is unknown)\n",
    "id": "coapp_failure_a12ed60b-fb4e-4080-b550-7db836d515c7",
    "downloadable_id": "downloadable_382775241565251"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://iconnectfx.com/view/54e72282-6617-ef11-996f-0050568299de/en",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : dyld: cannot load 'ffmpeg' (load command 0x80000034 is unknown)\n",
    "id": "coapp_failure_a12ed60b-fb4e-4080-b550-7db836d515c7",
    "downloadable_id": "downloadable_382775241565251"
  }
}

**Comment by paulrouget** [comment]
That's new. Can you go in the `settings > more settings` click on the `copy` button (top right corner of the settings panel) and paste the info here?

**Comment by HALLANM** [comment]
version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:109.0) Gecko/20100101 Firefox/115.0
{
    "http_media_download_strategy": "inbrowser",
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "/Volumes/RED_PILL/Users/MASTERmartin/dwhelper",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_wide_ui": true,
    "use_sidebar": false,
    "open_count_store": 12
}


Regards,

             Heath A. Martin
208-886-2457HM                     208-727-7249CELL
                        ***@***.*** ***@***.***>

**Comment by paulrouget** [comment]
What happens if you open the CoApp manually? Just open you Application folder and click on it.

- It should do nothing

If you see an error, please tell me.

**Comment by HALLANM** [comment]
It does nothing. . .never has.

Regards,

             Heath A. Martin
208-886-2457HM                     208-727-7249CELL
                        ***@***.*** ***@***.***>

**Reply by paulrouget** [reply]
ok (that's good).

Can you open the application "Terminal" and type this commend:

```
/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp --info
```

And tell me what you see.

**Comment by HALLANM** [comment]
"id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Volumes/RED_PILL/Users/MASTERmartin",
  "converterError": "Exit without answer”

Regards,

             Heath A. Martin
208-886-2457HM                     208-727-7249CELL
                        ***@***.*** ***@***.***>

**Comment by HALLANM** [comment]
I have not heard anything more from you on this. . .however. . .I am wondering if it is a SITE specific problem.  I have downloaded	several items from a different site with NO problem and then again, went to an altogether different site and had this same error.  

There must be something that some sites are doing to prevent downloading that you have not yet discovered or allowed for????

I can send you the details on this other failure if you desire.

Regards,

HM

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.

I visited the page identified in your original post above in page_url.  VDH downloaded a video of duration 4:03 no problem.  It played fine in VLC with both audio & video, although I did not sit & watch it, just sampled it at intervals.  It seemed to be fine.

Maybe this is yet another case of Firefox doing a better job than Chrome on a given item.  It wouldn't be the first time.  But I'm just a hack.  (A hack.  Not a hacker.  A hack.)  Paul is the developer.  I'd prioritize anything he says over anything I say.

**Comment by HALLANM** [comment]
Thank you.  I appreciate the input.

This might also be another situation where Windows doesn’t have a problem where a MAC does.  I have run into this before. . .I believe with this and other plug-ins. . .I couldn’t download on MAC but could on PC.

Next time I am on my PC, I will check to see if I can download from this site from there. . .that would definitely narrow down the issue.

BUT. . .I NEVER used to have a problem downloading from this site with VDH, on the MAC and the other site that I am NOW having issues with, I don’t think I ever had before an issue.


Regards,

             Heath A. Martin
208-886-2457HM                     208-727-7249CELL
                        ***@***.*** ***@***.***>

**Comment by HALLANM** [comment]
BTW. . for whatever it is worth. . .I tried accessing the problematic sites from my PC, with the new version of VDH and there was zero problems.  So, as I said. . .this appears too be a problem with the MAC version of VDH, the MAC version of Firefox, or a combo of both.

Thanks,
HM
